app/dashboard/page.tsx

```diff
@@ -19,19 +20,19 @@ export default function DashboardPage() {
     }
   }, [isAuthLoading, user, router])
 
-  const handleRefresh = async () => {
-    await supabase.auth.refreshSession();
-  }
-
   const handleLogout = async () => {
     try {
       setIsLoading(true);
 
       await logoutMutation.mutateAsync();
       await supabase.auth.signOut();
+      
+      // Clear any stored device nonce
+      localStorage.removeItem('deviceNonce');
```

Comment:
Not sure if it matters for the test app, but on the Wander Embed App we don't do this. The device nonce is set when the user first loads the app and it's only ever re-created if the user clears the browser data, but we never delete it.